godotproject/enemy-Ant.gd

```diff
@@ -17,11 +17,15 @@ var direction = -1
 var attackDamage = 1
 var attackState = STATE.idle_e
 var is_dead = false
+
+var tongue_can_damage = true
+var health = 100
 	
 func dead():
 	is_dead = true
 	velocity = 0
-	$AnimatedSprite.play("dead")
+#	$AnimatedSprite.play("dead")
```

Comment:
does this animation exist? If so can we add it in to the FGTS-81 PR?

Reply:
Removed for now